### PR TITLE
Misc link and other fix

### PR DESCRIPTION
**Pull request by Grummfy**
several fixes for building

### CODE REVIEW

File: source/en/written_help/annotation-extension.inc.rst [hunk 1]
```diff
@@ -3,9 +3,7 @@
 PHP Extensions
 ****************
 
-Some of your tests may require one or more PHP extension(s). Telling atoum that a test requires an extension is easily done through annotations and the tag @extensions.
-
-After the tag @extensions, just add one or more extension names, separated by a space.
+Some of your tests may require one or more PHP extension(s). Telling atoum that a test requires an extension is easily done through annotations and the tag @extensions. After the tag @extensions, just add one or more extension names, separated by a space.
```
Comment (jubianchi) [inline source/en/written_help/annotation-extension.inc.rst]: the `@extensions` might be in a code block

**Comment by jubianchi** [comment]
sems good :+1:

File: source/en/written_help/annotation-extension.inc.rst (hunk 1 above)
Comment (Grummfy) [inline source/en/written_help/annotation-extension.inc.rst]: fixed

**Comment by marmotz** [comment]
Thanks to @grummfy and @jubianchi

**Pull request merged**